doc/source/ray-air/examples/convert_existing_pytorch_code_to_ray_air.ipynb

```diff
@@ -59,11 +59,14 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "db36ae56",
    "metadata": {},
    "source": [
-    "Then we download the data:"
+    "Then we download the data: \n",
+    "\n",
+    "Assumption for this tutorial: your existing code is using the `torchvision.datasets` native to PyTorch. This tutorial continues to use `torchvision.datasets` to allow you to make as few code changes as possible. **Everything in this tutorial is also possible if you choose to use Ray Data, and you will also get the benefits of efficient preprocessing and multi-worker batch prediction.** See [here](train-datasets) for resources to get started with Ray Data."
```

Comment:
1. It can be any PyTorch DataLoader, not necessarily torchvision datasets
2. The benefit is for parallel preprocessing-- you can still use Ray Data for batch prediction without using for training (as this tutorial already does)
3. Maybe link to this tutorial instead: https://docs.ray.io/en/latest/ray-air/examples/torch_image_example.html?

Reply:
Thanks for the points! I have modified the description accordingly.